Paws4Me/Paws4Me/Sign In User/View Models/SignInViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ViewModels.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/24.
+//
+
+import Foundation
+import Firebase
```

Comment:
Do you need this here? 😄

Reply:
No I do not. I only need it inside my repository. Poor code clean up on my side. Apologies